Incompatible with gmock 1.8.1

Now fails to build on Arch GNU/Linux, bionic branch.
```
In file included from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.h:21,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.cpp:17:
/home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/gmock_fixes.h: In instantiation of 'static testing::internal::ActionResultHolder<std::unique_ptr<_Tp> >* testing::internal::ActionResultHolder<std::unique_ptr<_Tp> >::PerformDefaultAction(const testing::internal::FunctionMockerBase<F>*, const typename testing::internal::Function<F>::ArgumentTuple&, const string&) [with F = std::unique_ptr<mir::time::Alarm>(mir::LockableCallback*); T = mir::time::Alarm; typename testing::internal::Function<F>::ArgumentTuple = std::tuple<mir::LockableCallback*>; testing::internal::string = std::__cxx11::basic_string<char>]':
/usr/include/gmock/gmock-spec-builders.h:1542:46:   required from 'testing::internal::UntypedActionResultHolderBase* testing::internal::FunctionMockerBase<F>::UntypedPerformDefaultAction(void*, const string&) const [with F = std::unique_ptr<mir::time::Alarm>(mir::LockableCallback*); std::__cxx11::string = std::__cxx11::basic_string<char>]'
/usr/include/gmock/gmock-spec-builders.h:1538:42:   required from here
/home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/gmock_fixes.h:81:12: error: cannot bind rvalue reference of type 'testing::internal::RvalueRef<std::tuple<mir::LockableCallback*> >::type' {aka 'std::tuple<mir::LockableCallback*>&&'} to lvalue of type 'const ArgumentTuple' {aka 'const std::tuple<mir::LockableCallback*>'}
     return new ActionResultHolder(
            ^~~~~~~~~~~~~~~~~~~~~~~
         func_mocker->PerformDefaultAction(args, call_description));
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gmock/gmock-generated-function-mockers.h:44,
                 from /usr/include/gmock/gmock.h:62,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/gmock_fixes.h:39,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.h:21,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.cpp:17:
/usr/include/gmock/gmock-spec-builders.h:1511:10: note:   initializing argument 1 of 'testing::internal::FunctionMockerBase<F>::Result testing::internal::FunctionMockerBase<F>::PerformDefaultAction(typename testing::internal::RvalueRef<typename testing::internal::Function<F>::ArgumentTuple>::type, const string&) const [with F = std::unique_ptr<mir::time::Alarm>(mir::LockableCallback*); testing::internal::FunctionMockerBase<F>::Result = std::unique_ptr<mir::time::Alarm>; typename testing::internal::RvalueRef<typename testing::internal::Function<F>::ArgumentTuple>::type = std::tuple<mir::LockableCallback*>&&; std::__cxx11::string = std::__cxx11::basic_string<char>]'
   Result PerformDefaultAction(
          ^~~~~~~~~~~~~~~~~~~~
In file included from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.h:21,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.cpp:17:
/home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/gmock_fixes.h: In instantiation of 'static testing::internal::ActionResultHolder<std::unique_ptr<_Tp> >* testing::internal::ActionResultHolder<std::unique_ptr<_Tp> >::PerformDefaultAction(const testing::internal::FunctionMockerBase<F>*, const typename testing::internal::Function<F>::ArgumentTuple&, const string&) [with F = std::unique_ptr<mir::time::Alarm>(const std::function<void()>&); T = mir::time::Alarm; typename testing::internal::Function<F>::ArgumentTuple = std::tuple<const std::function<void()>&>; testing::internal::string = std::__cxx11::basic_string<char>]':
/usr/include/gmock/gmock-spec-builders.h:1542:46:   required from 'testing::internal::UntypedActionResultHolderBase* testing::internal::FunctionMockerBase<F>::UntypedPerformDefaultAction(void*, const string&) const [with F = std::unique_ptr<mir::time::Alarm>(const std::function<void()>&); std::__cxx11::string = std::__cxx11::basic_string<char>]'
/usr/include/gmock/gmock-spec-builders.h:1538:42:   required from here
/home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/gmock_fixes.h:81:12: error: cannot bind rvalue reference of type 'testing::internal::RvalueRef<std::tuple<const std::function<void()>&> >::type' {aka 'std::tuple<const std::function<void()>&>&&'} to lvalue of type 'const ArgumentTuple' {aka 'const std::tuple<const std::function<void()>&>'}
     return new ActionResultHolder(
            ^~~~~~~~~~~~~~~~~~~~~~~
         func_mocker->PerformDefaultAction(args, call_description));
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gmock/gmock-generated-function-mockers.h:44,
                 from /usr/include/gmock/gmock.h:62,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/gmock_fixes.h:39,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.h:21,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/qtmir-git/src/qtmir/tests/framework/mock_main_loop.cpp:17:
/usr/include/gmock/gmock-spec-builders.h:1511:10: note:   initializing argument 1 of 'testing::internal::FunctionMockerBase<F>::Result testing::internal::FunctionMockerBase<F>::PerformDefaultAction(typename testing::internal::RvalueRef<typename testing::internal::Function<F>::ArgumentTuple>::type, const string&) const [with F = std::unique_ptr<mir::time::Alarm>(const std::function<void()>&); testing::internal::FunctionMockerBase<F>::Result = std::unique_ptr<mir::time::Alarm>; typename testing::internal::RvalueRef<typename testing::internal::Function<F>::ArgumentTuple>::type = std::tuple<const std::function<void()>&>&&; std::__cxx11::string = std::__cxx11::basic_string<char>]'
   Result PerformDefaultAction(
          ^~~~~~~~~~~~~~~~~~~~
```

--- a/tests/framework/gmock_fixes.h
+++ b/tests/framework/gmock_fixes.h
@@ -76,10 +76,10 @@ class ActionResultHolder<std::unique_ptr<T>>
   template <typename F>
   static ActionResultHolder* PerformDefaultAction(
       const FunctionMockerBase<F>* func_mocker,
-      const typename Function<F>::ArgumentTuple& args,
+      typename Function<F>::ArgumentTuple args,
       const string& call_description) {
     return new ActionResultHolder(
-        func_mocker->PerformDefaultAction(args, call_description));
+        func_mocker->PerformDefaultAction(std::move(args), call_description));
   }
 
   // Performs the given action and returns the result in a new-ed

